draft-ietf-dprive-dnsoquic.md

```diff
@@ -308,14 +308,30 @@ DOQ_NO_ERROR (0x00):
 : No error.  This is used when the connection or stream needs to be closed, but
   there is no error to signal.
 
-DOQ_INTERNAL_ERROR (0x01):
+DOQ_REQUEST_CANCELLED (0x01):
+: A DoQ client uses this to signal that it wants to cancel an 
+outstanding transaction.
+
+DOQ_INTERNAL_ERROR (0x02):
 : The DoQ implementation encountered an internal error and is incapable of
   pursuing the transaction or the connection.
 
-DOQ_PROTOCOL_ERROR (0x02):
+DOQ_PROTOCOL_ERROR (0x03):
 : The DoQ implementation encountered an protocol error and is forcibly aborting 
   the connection.
 
+### Transaction Cancellation
+
+In QUIC, sending STOP_SENDING requests that a peer cease transmission on a
+stream. If a DoQ client wishes to cancel an outstanding request, it may issue a
+QUIC Stop Sending with error code DOQ_REQUEST_CANCELLED. The corresponding
+transaction MUST be abandoned. Servers MAY limit the number of
+DOQ_REQUEST_CANCELLED errors received on a connection before choosing to close
+the connection.
+
```

Comment:
What happens if the client sends a STOP_SENDING request with an error code different than DOQ_REQUEST_CANCELLED? What happens if the server sends a STOP_SENDING request?

Reply:
Reading this again. The text is good enough.